PYME/IO/MetaDataHandler.py

```diff
@@ -746,22 +746,23 @@ def getEntryNames(self):
 
 
 class OMEXMLMDHandler(XMLMDHandler):
-    _OME_UNITS_TO_UM = {'m': 1e6, 'mm': 1e3, 'um': 1.0, 'nm': 1e-3}
+    _OME_UNITS_TO_UM = {'m': 1e6, 'mm': 1e3, 'um': 1.0, ('%sm'%chr(181)): 1.0, 'nm': 1e-3}
```

Comment:
```suggestion
    _OME_UNITS_TO_UM = {'m': 1e6, 'mm': 1e3, 'um': 1.0, u'\u00B5m': 1.0, 'nm': 1e-3}
```

Using the unicode literal string is a bit cleaner and safer (much more likely to work across python versions and GUI toolkits etc ...) - unicode 181 is mu wheras ascii chr(181) is ?

Reply:
note that unicode literals are in hex, not decimal (hence the b5)